tests/console-backend-service/internal/domain/k8s/secret_test.go

```diff
@@ -211,20 +224,20 @@ func fixSecretsQuery() *graphql.Request {
 	return req
 }
 
-func fixSecretSubscription() *graphql.Request {
-	query := `subscription ($namespace: String!) {
-				secretEvent(namespace: $namespace) {
-					type
-					secret {
-						name
-					}
-				}
-			}`
-	req := graphql.NewRequest(query)
-	req.SetVar("namespace", secretNamespace)
-
-	return req
-}
+//func fixSecretSubscription() *graphql.Request {
```

Comment:
Why is it commented?

Reply:
We are disabling secret subscriptions temporarily due to security reasons.